src/app/components/shared/details-fields/details-fields.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { DetailsFieldsComponent } from './details-fields.component';
+
+describe('DetailsFieldsComponent', () => {
+  let component: DetailsFieldsComponent;
+  let fixture: ComponentFixture<DetailsFieldsComponent>;
+
+  beforeEach(async(() => {
```

Comment:
should not we have a test checking the details fields appear when a project is selected.
what about another test checking the clock-in button appears when the mouse is over a project?